test/acceptance/page-objects/claimJourney-page.js

```diff
@@ -130,33 +175,92 @@ class StartPageActions extends CommonActions {
     await this.sendKey(DIFFERENT_YEAR,year)
   }
 
-  async VerifyError_PastDate() {
-    await this.clickOn(ANOTHER_DAY_BUTTON)
-    const currentDate = new Date();
-    const day = currentDate.getDate() + 1;
-    const month = currentDate.getMonth() + 1;
-    const year = currentDate.getFullYear();
-    await this.sendKey(DIFFERENT_DAY, day)
-    await this.sendKey(DIFFERENT_MONTH, month)
-    await this.sendKey(DIFFERENT_YEAR, year)
+  async dateFormat(){
+    const date_Value= this.inputCurrentDate('correct');
+    const arrayOfDate=(await date_Value).split(",");
+    another_day=arrayOfDate[0]
+    another_month=arrayOfDate[1]
+    another_year=arrayOfDate[2]
   }
-  async VerifyError_BeforeApplicationDate() {
+  async clickOnAnotherDay(){
     await this.clickOn(ANOTHER_DAY_BUTTON)
-    const currentDate = new Date();
-    const day = currentDate.getDate() + 1;
-    const month = currentDate.getMonth() - 1;
-    const year = currentDate.getFullYear();
-    await this.sendKey(DIFFERENT_DAY, day)
-    await this.sendKey(DIFFERENT_MONTH, month)
-    await this.sendKey(DIFFERENT_YEAR, year)
+    await this.sendKey(DIFFERENT_DAY,another_day);
+    await this.sendKey(DIFFERENT_MONTH,another_month);
+    await this.sendKey(DIFFERENT_YEAR,another_month);
+     }
+
+  async verifyDate_Error(day) {
+       await this.clickOn(ANOTHER_DAY_BUTTON)
+       await this.dateFormat();
+   if(day=='future'){
+    await this.sendKey(DIFFERENT_DAY,parseInt(another_day)+1)
+    await this.sendKey(DIFFERENT_MONTH,parseInt(another_month)+1)
+    await this.sendKey(DIFFERENT_YEAR, another_year)
+       }
+       else{
+    await this.sendKey(DIFFERENT_DAY, parseInt(another_day)+1)
+    await this.sendKey(DIFFERENT_MONTH,parseInt(another_month)-1)
+    await this.sendKey(DIFFERENT_YEAR,parseInt(another_year)-1)
+    
   }
+  }
+
+  // async VerifyError_PastDate() {
+  //   await this.clickOn(ANOTHER_DAY_BUTTON)
+  //   const currentDate = new Date();
+  //   const day = currentDate.getDate() + 1;
+  //   const month = currentDate.getMonth() + 1;
+  //   const year = currentDate.getFullYear();
+  //   await this.sendKey(DIFFERENT_DAY, day)
+  //   await this.sendKey(DIFFERENT_MONTH, month)
+  //   await this.sendKey(DIFFERENT_YEAR, year)
+  // }
+  // async VerifyError_BeforeApplicationDate() {
+  //   await this.clickOn(ANOTHER_DAY_BUTTON)
+  //   const currentDate = new Date();
+  //   const day = currentDate.getDate() + 1;
+  //   const month = currentDate.getMonth() - 1;
+  //   const year = currentDate.getFullYear();
+  //   await this.sendKey(DIFFERENT_DAY, day)
+  //   await this.sendKey(DIFFERENT_MONTH, month)
+  //   await this.sendKey(DIFFERENT_YEAR, year)
+  // }
```

Comment:
What is this for?

Reply:
this is for error message validation for date

Reply:
I mean why is it commented out? If we don't want it then we should remove it or if we are saving it for the future it would be good to understand why